eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/QueryWebHookConfigByIdHandler.java

```diff
@@ -60,7 +61,7 @@ public void handle(HttpExchange httpExchange) throws IOException {
 
         try (OutputStream out = httpExchange.getResponseBody()) {
             WebHookConfig result = operation.queryWebHookConfigById(webHookConfig); // operating result
-            out.write(JsonUtils.toJSONString(result).getBytes(Constants.DEFAULT_CHARSET));
+            out.write(Objects.requireNonNull(JsonUtils.toJSONString(result).getBytes(Constants.DEFAULT_CHARSET)));
```

Comment:
I think it should be done as follows:
`out.write(Objects.requireNonNull(JsonUtils.toJSONString(result)).getBytes(Constants.DEFAULT_CHARSET));`